cms/static/sass/views/_outline.scss

```diff
@@ -640,10 +640,24 @@
     color: theme-color("primary");
   }
 
+  .number-highlights {
+    background: $uxpl-blue-base;
```

Comment:
I don't think this variable is defined. I'm getting an error about this in my devstack's `make studio-static`. It looks like `$uxpl-light-blue-base` exists though.

Reply:
hmm.. Glad I rebased with master just now.  `$uxpl-blue-base` existed until a few moments ago.  The intent was for it to be the same color as the `.highlights-button` declared above - which Andy _just_ changed in a recent PR to `theme-color("primary")`.  I'll update now.